### DISCUSSION TITLE
Adding ```font_family``` as an optional parameter for the ```add_legend``` function

### DISCUSSION THREAD

**Discussion by ZAARAOUI999** [pyvista/pyvista#5394] [Show and tell]
Hi everyone, I have a small suggestion about the ```add_legend``` function, and it's to add an optional parameter for customizing font family of legend's text, so I can choose my favourite one. This is what I get after modifiying the defined function:
```Before (legend default font):```
![arial](https://github.com/pyvista/pyvista/assets/115699524/44604fa4-5f92-49d5-83b3-7a66645bc634)

```After (courier font):```
![courier](https://github.com/pyvista/pyvista/assets/115699524/1cde112a-5603-4da1-ade1-2c9d4dd7af14)

```The modification:```
```python
 def add_legend(
        self,
        labels=None,
        bcolor=(0.5, 0.5, 0.5),
        border=False,
        size=(0.2, 0.2),
        name=None,
        loc='upper right',
        face='triangle',
        font_family='courier',
    ):
    ...
        ###################### - Added - ########################
        legend_text = self._legend.GetEntryTextProperty()
        legend_text.SetFontFamily(parse_font_family(font_family))
        #########################################################

        self.add_actor(self._legend, reset_camera=False, name=name, pickable=False)
        return self._legend
```

**Comment by banesullivan** [comment]
Please feel welcome to contribute this new feature! We love community contributions and would be happy to assist you in any Pull Request for this.

Take a look at our contributing guide for where to start: https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst

**Reply by ZAARAOUI999** [reply]
So, I still need your help 🤔 (after reading the contributing guide). Can anyone else add the ```font_family``` parameter to the specified function?, I will be so thankful 🥰.

**Reply by tkoyama010** [reply]
@ZAARAOUI999 I reflected your code in #5396. Please check to see if it meets your requirements.

**Reply by ZAARAOUI999** [reply]
Exactly that's what I need. Now legend actor can detect global theme font 💪. Thanks 🙂.
